src/kirby/components/list/list.component.ts

```diff
@@ -7,7 +7,10 @@ import {
   OnChanges,
   Output,
   TemplateRef,
+  ViewChild,
 } from '@angular/core';
+import { RadListViewComponent } from 'nativescript-ui-listview/angular/listview-directives';
+import { isIOS } from 'tns-core-modules/platform';
```

Comment:
Try to run a build for web. And you will see it fails 😄 

You can not import any from {N} specific modules in a shared .ts file. You will have to create a `.tns` version of the component to do this. Or a service with both a `.ts` and a `.tns.ts` version.

Reply:
Yes, will use the platform service instead :)